tests/test_frontend_app_helper_modules.py

```
@@ -0,0 +1,701 @@
+
+import shutil
+import subprocess
+from pathlib import Path
+
+import pytest
+
+REPO_ROOT = Path(__file__).resolve().parent.parent
+
+
+def run_node_module(script: str) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)